draft-dekater-scion-controlplane.md

```diff
@@ -1461,22 +1485,111 @@ When the segment-request handler of a *core AS* control service receives a path
 
 # Security Considerations
 
-One of the fundamental objectives that guided the design of SCION is security, in particular network security. See chapter 7 of the SCION book (Security Analysis), which states the precise security goals of various network participants and how SCION achieves these goals in the presence of different types of adversaries {{CHUAT22}}.
+As described previously, the goal of SCION’s beaconing process in the control plane is to discover paths between any two ASes and make them available to other ASes. This section describes the possible security risks and attacks that SCION's control plane may be prone to, and how these may be mitigated. The focus lies on *inter*-AS routing: SCION does not solve intra-AS routing issues, nor does it provide end-to-end payload encryption, and identity authentication. These topics lie therefore outside the scope of this section.
+
+**Note:** This section only discusses SCION control plane- and routing-specific security considerations. For security considerations related to the SCION control-plane PKI, see {{I-D.scion-cppki}}. {{I-D.scion-dp}} includes security considerations that concern the SCION data plane and data forwarding.
+
+This section focuses on three kinds of security risks in the control plane. The first risk is when an adversarial entity (e.g., a government) controls one or all core ASes of an ISD and tries to frustrate the beaconing process from the top down (see [](#topdown-frust)). Also "ordinary" (non-core) adversaries that try to manipulate the beaconing process pose a risk to the control plane (see [](#manipulate-beaconing)). The third kind of security risks are Denial of Services (DoS) attacks, where attackers overload different parts of the IT infrastructure (see [](#dos-cp)).
```

Comment:
I'm not sure 'frustrate' is the right term here, but I need to understand the context better.